.cruft.json

```
@@ -0,0 +1,25 @@
+{
+  "template": "https://github.com/appuio/commodore-component-template.git",
+  "commit": "3e1ac242633b3345621ba9f2b72f8dfea8e6ac4a",
+  "checkout": "main",
+  "context": {
+    "cookiecutter": {
+      "name": "Openshift4 Networking",
```

Usually, if we use a nice display name for OCP4 components, we use "OpenShift 4"

```suggestion
      "name": "OpenShift 4 Networking",
```

---

Should that also be:
```
"checkout": "master",
```

---

Ah no, thats for the template.